sdk/go/redirects.go

```diff
@@ -35,19 +35,10 @@ func (cli *Client) GetRedirect(ctx context.Context, options Options, key string)
 }
 
 // GetRedirects gets the /redirects endpoint
-func (cli *Client) GetRedirects(ctx context.Context, options Options, count, cursor string) (*models.Redirects, apiError.Error) {
+func (cli *Client) GetRedirects(ctx context.Context, options Options) (*models.Redirects, apiError.Error) {
```

Comment:
So I really dislike the idea of using `http.Header` and `url.Values` as parameters for the sdk. That then couples the API of the client to the protocol subsequently used by the client. This is not a precedent you've introduced so I won't address it here. But I will mention it to Linden separately. I notice that the search API example he sent you in his comment does the same thing.